src/azure-cli-core/setup.py

```diff
@@ -53,7 +53,7 @@
     'jmespath',
     'knack~=0.11.0',
     'msal-extensions~=1.0.0',
-    'msal[broker]==1.27.0',
+    'msal[broker]==1.26.0',
```

Comment:
MSAL 1.27.0 raised the upper bound of `pymsalruntime` (https://github.com/AzureAD/microsoft-authentication-library-for-python/commit/59c3000192e92a49f483045071b97aa79929d19f), but the latest `pymsalruntime` 0.14.1 fails with:

```
Called AcquireTokenInteractively but no redirect URI was provided. Status: Response_Status.Status_ApiContractViolation, Error code: 0, Tag: 590899094
```